app/views/jobs/show.html.erb

```diff
@@ -26,7 +26,7 @@
             <li><strong>Budget: </strong> <%= @job.budget %></li>
             <li><strong>Status: </strong> <%= @job.status %></li>
           </ul>
-          <u>Ratings:</u>
+          <u>Reviews:</u>
```

Comment:
There should be only one review per job, so Review (no s), or you can use pluralize